benches/benchmarks/check_texts.rs

```diff
@@ -29,12 +31,12 @@ async fn request_until_success(req: &Request, client: &ServerClient) -> Response
 }
 
 #[tokio::main]
-async fn check_text_basic(text: &str) -> Response {
+async fn check_text_basic(text: &'static str) -> Response {
     let client = ServerClient::from_env().expect(
         "Please use a local server for benchmarking, and configure the environ variables to use \
          it.",
     );
-    let req = Request::default().with_text(text.to_string());
+    let req = Request::default().with_text(Cow::Borrowed(text));
```

Comment:
Same comment as above
```suggestion
    let req = Request::default().with_text(text);
```